ga4gh/backend.py

```diff
@@ -164,13 +164,19 @@ def _search(self, start, end):
 
     @classmethod
     def _getStart(cls, readAlignment):
-        return readAlignment.alignment.position.position
+        if readAlignment.alignment is None:
+            return -2
```

Comment:
If this is a coordinate-sorted input file, these interspersed ones are almost certainly unmapped reads that have been sorted next to their mates.  See [SAMv1.pdf](http://samtools.github.io/hts-specs/SAMv1.pdf) §2 4.1.  (BTW both-ends-unmapped read pairs (§2 4.2) appear at the _end_ of a sorted file.)